db/l2db/l2db.go

```diff
@@ -319,9 +319,12 @@ func (l2db *L2DB) GetPendingUniqueFromIdxs() ([]common.Idx, error) {
 	return idxs, nil
 }
 
+const invalidateOldNoncesInfo = `Nonce is smaller or equal than account nonce`
```

Comment:
I think it should just be `smaller than`. See the SQL query: `tx_pool.nonce < updated_acc.nonce;`